.github/workflows/ci.yaml

```diff
@@ -50,12 +65,44 @@ jobs:
             ~/.cargo/git/db/
             target/
           key: ${{ runner.os }}-cargo-release-${{ hashFiles('**/Cargo.lock') }}
-      - name: Build amd64 Executables
+      - name: Build Executables
+        run: cargo build --release --target ${{ matrix.platform.target }} ${{ matrix.platform.build-config }}
```

Comment:
Now that has moved away what we usually do in local which is release. Some users refer to CI to see how they build docker. Anyway it's an opinionated approach if I say repeat is better than matrix. So feel free to discard my opinion :)

Reply:
then it does not make sense to do it in matrix way, and also in parallel, because this in such way it will be even harder to extend it

Reply:
and sure, it can be done, even better, but we should try to do it step by step, when next platform will be needed, or something else change, which will require noticeable changes in CI

Reply:
Sure. I understand your concern. I wouldn't do it this way personally, and it is much easier for me to copy paste than understanding the nuances of when to use `if statements` for which platform and having to figure out the commonalities of these builds.

Reply:
currently it is 1 conditional step from 10, 
if it was 5 from 10, it could make sense to do copy/paste style, 
but with 1, it does not make sense to me to do it like that